modules/ROOT/pages/dw-core-functions-splitby.adoc

```diff
@@ -113,6 +113,34 @@ output application/json
 "192"
 ----
 
+=== Example
+
+This example uses a Java regular expression to split a string by `.` at every point the input string matches the regex.
```

Comment:
Maybe it's worth mentioning that the regex doesn't consider `.` inside `.

Reply:
I updated it.